5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -79,15 +94,17 @@ EOSQL
 
   # Do not care what option is compulsory here, just create what is specified
   if [ -v MYSQL_USER ]; then
+    log_info "Creating user ${MYSQL_USER} ..."
```

Comment:
"Creating user specified by MYSQL_USER (${MYSQL_USER)"

so users will know from where we get that user?